rubin_sim/scheduler/utils/sky_area.py

```diff
@@ -30,8 +30,10 @@ class Sky_area_generator:
     ----------
     nside : `int`
         Healpix resolution for map
+    XXX:  Moved magic values to kwargs. Should refactor to eliminate nvis_wfd_default.
```

Comment:
I can see moving everything from nvis_wfd_default into just maps for each region, scaled 0-1 .. and then having max weights in each region be the fraction for that region (i.e. the NES would not have a 'map' that goes 0-1, but instead 0-0.3, etc.). 
It might make scaling the filter balances more difficult, and we would drop integers in the 'calculate visits' parts, but those seem reasonable. 

I guess it's fine to leave this as-is for now and update later, but it would also be reasonable to update now and then re-run the baseline with the actual 'final' version, if that's something that we would be doing on a short timeline anyway. (I'd rather not run half of v2.0 with the changed version and half without, I guess).